### PR TITLE
Add new relic support on CI

### PR DESCRIPTION
**Pull request by piggito**
New relic is useful for monitoring and profiling apps so it would be useful to implement them on CI to retrieve stats about performance.

### CODE REVIEW

**Comment by andypost** [comment]
Please squash commits

File: docker/docker-compose.override.yml.ci [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      NEW_RELIC_LICENSE_KEY: "${NEW_RELIC_LICENSE_KEY}"
+      NEW_RELIC_APPNAME: "${COMPOSE_PROJECT_NAME}"
+    volumes:
+      - "./90-mail.ini:/etc/php7/conf.d/90-mail.ini:z"
+#    depends_on:
+#      - mysql
+# Uncomment next line if you need PHP XDebug.
+#    command: php-fpm7 -F -d zend_extension=xdebug.so
+
+  newrelic:
```
Comment (andypost) [inline docker/docker-compose.override.yml.ci:18]: instead of that CI could use extra file with just newrelic chnages

File: docker/docker-compose.override.yml.default [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "2.1"
+version: "2.2"
```
Comment (andypost) [inline docker/docker-compose.override.yml.default:1]: this changes better to move to separate PR

**Review by andypost — CHANGES_REQUESTED** [review]
Needs to split upgrade to 2.2 and CI additions via `.env` change (extend first line)

File: docker/docker-compose.override.yml.default (hunk 2 above)
Comment (andypost) [reply docker/docker-compose.override.yml.default:1]: Filed https://github.com/skilld-labs/skilld-docker-container/pull/286

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Variables to add in Gitalb UI : Settings > CI/CD
 # - SONAR_HOST // Optional
 # - SONAR_TOKEN // Optional
+# - NEW_RELIC_LICENSE_KEY
```
Comment (davidferlay) [inline .gitlab-ci.yml:4]: Can you please change to `NEW_RELIC_LICENSE_KEY  // Optional` if so ?

File: Makefile [hunk 4]
```diff
@@ -74,6 +74,7 @@ endif
 	$(call php-0, apk add --no-cache graphicsmagick tzdata $(ADD_PHP_EXT))
 	# Set up timezone
 	$(call php-0, cp /usr/share/zoneinfo/Europe/Paris /etc/localtime)
+	make -s newrelic
```
Comment (davidferlay) [inline Makefile:77]: Maybe best to not add in make all by default ? cause newrelic requires registration etc

Comment (andypost) [reply Makefile:78]: Added comment - it will work only in `NEW_RELIC_LICENSE_KEY=xxx make provision` will be fired or when in CI environment variable defined

Comment (davidferlay) [reply Makefile:78]: So NEW_RELIC_LICENSE_KEY should be added to .env.default (or at least documented in README.md)

Comment (andypost) [reply Makefile:78]: Document ++ because it needed only on RA
For local we have xhprof and blackfire

Comment (andypost) [reply Makefile:78]: Fixed by changing readme bellow, moreover sharing a key/secret via `.env` was initially bad idea

**Comment by andypost** [comment]
@davidferlay thank you!
